### PR TITLE
Add Chinese Translations for Alda Documention

### PR DESCRIPTION
**Pull request by OWALabuy**
**Description:**
This PR adds Chinese translations for four Alda documentation pages to increase accessibility for Chinese-speaking users. The translated documents are located in the `doc/doc_zh_cn` directory.

### Summary of Changes:
- Translated the following documents into Chinese:
  1. index.md
  2. notes.md
  3. tempo.md
  4. editor-plugins.md

### Motivation:
With this translation, Alda will be more accessible to a wider audience, potentially growing the user base and community. These translations are a first step, and I plan to continue translating more documentation.

### Notes:
- Let me know if there are any specific changes or adjustments needed for these translations.
- I’m open to feedback and willing to make any necessary revisions.

### CODE REVIEW

File: doc/doc_zh_cn/chords_zh_cn.md [hunk 1]
```diff
@@ -0,0 +1,11 @@
+# 和弦
+
+*此页面翻译自[chouds.md](../chouds.md)*
```
Comment (daveyarwood) [inline doc/doc_zh_cn/chords_zh_cn.md:3]: Possible typo?

File: doc/tempo.md [hunk 2]
```diff
@@ -55,7 +55,7 @@ measure, where each beat is a dotted quarter note.
 
 At a certain point in the piece, you want to transition into a 3/2 section --
 still 3 beats per measure, but now each beat is a half note. You want the
-"pulse" to stay the same, but now each beat is subdivided into 4 half notes
+"pulse" to stay the same, but now each beat is subdivided into 4 eighth notes
```
Comment (daveyarwood) [inline doc/tempo.md:58]: Good catch!

File: player/gradle/wrapper/gradle-wrapper.properties [hunk 3]
```diff
@@ -1,5 +1,6 @@
+#Sun Sep 01 19:36:42 CST 2024
 distributionBase=GRADLE_USER_HOME
+distributionUrl=https\://services.gradle.org/distributions/gradle-7.4.2-bin.zip
 distributionPath=wrapper/dists
-distributionUrl=https\://services.gradle.org/distributions/gradle-6.8.3-bin.zip
-zipStoreBase=GRADLE_USER_HOME
 zipStorePath=wrapper/dists
+zipStoreBase=GRADLE_USER_HOME
```
Comment (daveyarwood) [inline player/gradle/wrapper/gradle-wrapper.properties:6]: Can we revert the changes to this file, since they aren't needed?

**Review by daveyarwood — CHANGES_REQUESTED** [review]
Looks great, thanks so much for doing this!

File: doc/doc_zh_cn/chords_zh_cn.md (hunk 1 above)
Comment (OWALabuy) [reply doc/doc_zh_cn/chords_zh_cn.md:3]: sorry... I will correct it

File: doc/doc_zh_cn/alda-2-migration-guide_zh_cn.md [hunk 4]
```diff
@@ -0,0 +1,256 @@
...
+]
+```
+
+如您所见，Alda 1 和 Alda 2 以不同的方式呈现相同的信息！
+如果您碰巧构建了任何依赖于 Alda 1 `alda parse` 输出的工具或工作流，则可能需要在升级到 Alda 2 后进行调整
+
+## 就是这样！
+
+<<<<<<< HEAD
+我们希望您喜欢 Alda 2！请随时加入我们的 [Slack 群组][alda-slack] 并让我们知道您的想法。如果你遇到 bug 或任何其他类型的奇怪行为，你也可以 [开issue][open-an-issue]，我们很乐意帮助您解决问题！
```
Comment (cewno) [inline doc/doc_zh_cn/alda-2-migration-guide_zh_cn.md:243]: `开 issue` 多用于日常交流中，在文档等较为正式语境下用`打开一个 issue`更好，且方便不熟悉开源社区的人使用

File: doc/doc_zh_cn/alda-2-migration-guide_zh_cn.md [hunk 5]
```diff
@@ -0,0 +1,256 @@
...
+=======
+我们希望您喜欢 Alda 2！请随时加入我们的 [Slack 群组][alda-slack] 并让我们知道您的想法。
+如果你遇到 bug 或任何其他类型的奇怪行为，你也可以 [打开一个 issue][open-an-issue]，我们很乐意帮助您解决问题！
+>>>>>>> 49cbb58fe0457925da96f47923b0833ea92a72b6
```
Comment (cewno) [inline doc/doc_zh_cn/alda-2-migration-guide_zh_cn.md:247]: 这里是不是没处理好冲突

**Review by cewno — CHANGES_REQUESTED** [review]

**Review by daveyarwood — APPROVED** [review]
@OWALabuy @cewno This is awesome! Thanks so much for your hard work on this.

**Pull request merged**